### PR TITLE
chore: address unused functions and constants

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Comment by hezhizhen** [comment]
@caoxianfei1 I add `//nolint:unused` to unused code, and you can comment on each one to keep it or delete it.

File: api/v1/curvecluster_types.go [hunk 1]
```diff
@@ -30,13 +30,13 @@ const (
...
-	ClusterPhaseReady ConditionType = "Ready"
+	ClusterPhaseReady ConditionType = "Ready" //nolint:unused
 	// ClusterPhaseDeleting indicates the cluster is running to delete.
 	ClusterPhaseDeleting ConditionType = "Deleting"
 	// ClusterPhaseError indicates the cluster created failed because of some reason.
-	ClusterPhaseError ConditionType = "Failed"
+	ClusterPhaseError ConditionType = "Failed" //nolint:unused
 	// ClusterPhaseUnknown is unknown phase
-	ClusterPhaseUnknown ConditionType = "Unknown"
+	ClusterPhaseUnknown ConditionType = "Unknown" //nolint:unused
```
Comment (caoxianfei1) [inline api/v1/curvecluster_types.go:39]: delete

File: api/v1/curvecluster_types.go [hunk 2]
```diff
@@ -30,13 +30,13 @@ const (
 	// ClusterPhasePending indicates the cluster is running to create.
 	ClusterPhasePending ConditionType = "Pending"
 	// ClusterPhaseReady indicates the cluster has been created successfully.
-	ClusterPhaseReady ConditionType = "Ready"
+	ClusterPhaseReady ConditionType = "Ready" //nolint:unused
 	// ClusterPhaseDeleting indicates the cluster is running to delete.
 	ClusterPhaseDeleting ConditionType = "Deleting"
 	// ClusterPhaseError indicates the cluster created failed because of some reason.
-	ClusterPhaseError ConditionType = "Failed"
+	ClusterPhaseError ConditionType = "Failed" //nolint:unused
```
Comment (caoxianfei1) [inline api/v1/curvecluster_types.go:37]: k

File: api/v1/curvecluster_types.go [hunk 3]
```diff
@@ -30,13 +30,13 @@ const (
 	// ClusterPhasePending indicates the cluster is running to create.
 	ClusterPhasePending ConditionType = "Pending"
 	// ClusterPhaseReady indicates the cluster has been created successfully.
-	ClusterPhaseReady ConditionType = "Ready"
+	ClusterPhaseReady ConditionType = "Ready" //nolint:unused
```
Comment (caoxianfei1) [inline api/v1/curvecluster_types.go:33]: k

File: pkg/config/variables.go [hunk 4]
```diff
@@ -57,6 +58,7 @@ const (
 	ROLE_METASERVER    = "metaserver"
 )
 
+//nolint:unused
```
Comment (caoxianfei1) [inline pkg/config/variables.go:61]: delete

File: pkg/config/variables.go [hunk 5]
```diff
@@ -66,6 +68,7 @@ type Var struct {
 }
 
 var (
+	//nolint:unused
```
Comment (caoxianfei1) [inline pkg/config/variables.go:71]: delete

File: pkg/etcd/spec.go [hunk 6]
```diff
@@ -185,6 +185,7 @@ func (c *Cluster) makeChmodDirInitContainer(etcdConfig *etcdConfig) v1.Container
 	return container
 }
 
+//nolint:unused
```
Comment (caoxianfei1) [inline pkg/etcd/spec.go:188]: delete

**Comment by caoxianfei1** [comment]
keep no comment.

**Comment by hezhizhen** [comment]
> keep no comment.

Done. PTAL

**Pull request merged**